src/main/java/at/ac/tuwien/kr/alpha/grounder/parser/ParseTreeVisitor.java

```diff
@@ -298,24 +320,24 @@ public Literal visitNaf_literal(ASPCore2Parser.Naf_literalContext ctx) {
 		// naf_literal : NAF? (external_atom | classical_literal | builtin_atom);
 		isCurrentLiteralNegated = ctx.NAF() != null;
```

Comment:
It seems that `isCurrentLiteralNegated` now can be a variable local to this method, since it is no longer passed around.

Reply:
Right, thanks! Implemented in a217ffd89f714badf89a5ba8700cf7da78557084.